### DISCUSSION TITLE
How I can span main block up to length of the column-margin?

### DISCUSSION THREAD

**Discussion by danieltomasz** [quarto-dev/quarto-cli#1469] [Q&A]
Currently I use the zipped code 
[side_by_side.qmd.zip](https://github.com/quarto-dev/quarto-cli/files/9150309/side_by_side.qmd.zip)

With this effect
![image](https://user-images.githubusercontent.com/7980381/179996170-cb57ac48-a97d-44cd-a5ac-0d5965d3b64e.png)

I would like to start second paragraph (quarto2) only after the  first margin block will ends.
How I can achieve that?
Or maybe there other ways for achievieng that, like columns?

**Comment by dragonstyle** [comment]
Right now, there isn't an automatic way to push that main body block down below the margin block - the margin blocks are allowed to overflow their parents in order to prevent unexpected gaps in the main body of documents when there is longer margin content. 

One idea - you could apply a `margin-bottom` to the first paragraph to manually position the second paragraph (pretty manual, but would work). There is CSS being applied to the margin elements which make them essentially 'heightless' (look at the `no-row-height` class)- overriding this CSS may give you the result you'd like (letting them be a natural height).
